portfolio/src/main/webapp/script.js

```diff
@@ -13,5 +13,14 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds interests to the portfolio page.
  */
+async function getServerInterest() {
+    const responseFromServer = await fetch("/interests");
+    // The json() function returns an object that contains fields that we can
+    // reference to create HTML.
+    const inJFile = await responseFromServer.json();
+    const interestListElement = document.getElementById('server-interests-container');
+    interestListElement.innerHTML = inJFile;// can I return a string json file here?
```

Comment:
Yes, you can, but it just looks a little basic, this completes week 2's work but you could try making this feature better by showing it in the same page (not sending to another page from your portfolio) and also just showing one random interest instead of the full list.

Reply:
<img width="478" alt="image" src="https://user-images.githubusercontent.com/35376654/174887450-1a2693b2-da03-478c-b7a8-d532345f5d21.png">

Reply:
First please complete week 3 tasks after merging this PR